src/Makefile

```diff
@@ -18,6 +18,7 @@ TARGET_SOURCES_TEST := memcachedstore.cpp \
                        snmp_row.cpp \
                        snmp_accumulator_table.cpp \
                        snmp_counter_table.cpp \
+					   snmp_success_fail_count_table.cpp \
```

Comment:
If you have `setlocal noexpandtab` in your `~/.vim/ftplugin/make.vim` it will prevent vim expanding tabs to spaces in *anywhere* in the file, even where it is allowed to. 

When adding a new file to these lists I usually just copy the preceding line and edit it. 

I also have the following in my vimrc to make tabs visible in the UI:

```
set list
set listchars=tab:>\ ,
```